multiapps-controller-persistence/src/main/resources/org/cloudfoundry/multiapps/controller/persistence/db/changelog/sql/create_unlink_file_content_triggers.sql

```diff
@@ -1,4 +1,4 @@
-CREATE TRIGGER DELETE_FROM_SYSTEM_TABLE_LM_SL_PERSISTENCE_FILE AFTER UPDATE OR DELETE
+CREATE TRIGGER DELETE_FROM_SYSTEM_TABLE_LM_SL_PERSISTENCE_FILE AFTER DELETE
```

Comment:
The same comment applies here. You can't modify existing Liquibase changes like this - you need to create new ones.